src/cljs/nr/deckbuilder.cljs

```diff
@@ -419,7 +424,7 @@
         best-card (lookup (:side card) card)]
     (if (js/isNaN qty)
       (swap! s assoc :quantity 3)
-      (let [max-qty (or (:limited best-card) 3)
+      (let [max-qty (or (:deck-limit best-card) 3)
```

Comment:
Could probably change this to `(:deck-limit best-card 3)` unless that breaks for some reason.

Reply:
I think it's written this way to protect against `{:deck-limit nil}`, but I agree that it's ugly. Seeing as we control the data now, it's probably safe to change those back to the standard way.